vanilla-javascript-demo/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "vanilla-javascript-demo",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Example of building ChemDraw JS application with JavaScript",
+  "scripts": {
+    "start": "serve",
+    "preinstall": "rm -rf asset",
```

Comment:
You can't use rm for this you will need to find a package with a cross platform folder deleting cli tool. There are many of these. I'm sure we use something like this in the CDJS scripts.